### PR TITLE
5 implement supplier poduct add

### PR DESCRIPTION
**Pull request by salinda-yasas-99**

### CODE REVIEW

File: src/app/supplier/products/add/page.js [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+      message.error("You can only upload up to 6 images!");
+    } else {
+      setFileList(newFileList);
+    }
+  };
+
+  const onFinish = async (values) => {
+    try {
+      if (!fileList || fileList.length !== 6) {
+        message.error("Please upload at least 6 images.");
```
Comment (lakshankd) [inline src/app/supplier/products/add/page.js:63]: remove this message

File: src/app/supplier/products/add/page.js [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+        return;
+      }
+
+      const seoTagsArray = values.seoTags
+        ?.split(",")
+        .map((tag) => tag.trim())
+        .filter((tag) => tag !== "");
+
+      if (!seoTagsArray || seoTagsArray.length === 0) {
+        message.error(
```
Comment (lakshankd) [inline src/app/supplier/products/add/page.js:73]: remove this alert

File: src/app/supplier/products/add/page.js [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+        stockQuantity: values.stockQuantity,
+        wholesalePrice: values.WholePrice,
+        retailPrice: values.RetailPrice,
+        seoTags: seoTagsArray,
+        image1: fileList[0].originFileObj,
+        image2: fileList[1].originFileObj,
+        image3: fileList[2].originFileObj,
+        image4: fileList[3].originFileObj,
+        image5: fileList[4].originFileObj,
+        image6: fileList[5].originFileObj,
```
Comment (lakshankd) [inline src/app/supplier/products/add/page.js:91]: make this an array

File: src/app/supplier/products/add/page.js [hunk 4]
```diff
@@ -0,0 +1,408 @@
...
+    }
+  };
+
+  useEffect(() => {
+    fetchCategories();
+  }, []);
+
+  const uploadButton = (
+    <button
+      style={{
```
Comment (lakshankd) [inline src/app/supplier/products/add/page.js:123]: move this to a separate css file (supplier-styles.css)

File: src/app/supplier/products/add/page.js [hunk 5]
```diff
@@ -0,0 +1,408 @@
...
+        border: 0,
+        background: "none",
+      }}
+      type="button"
+    >
+      <PlusOutlined />
+      <div
+        style={{
+          marginTop: 8,
+        }}
```
Comment (lakshankd) [inline src/app/supplier/products/add/page.js:133]: move this to a separate css file (supplier-styles.css)

File: src/app/supplier/products/add/page.js [hunk 6]
```diff
@@ -0,0 +1,408 @@
...
+      </Row>
+      <SellerPageContainer
+        childern={
+          <Row gutter={[16, 16]} justify="start">
+            <Col span={24}>
+              <Form
+                labelCol={{ span: 8 }}
+                wrapperCol={{ span: 16 }}
+                layout="horizontal"
+                style={{ maxWidth: 1000 }}
```
Comment (lakshankd) [inline src/app/supplier/products/add/page.js:159]: move this to a separate css file (supplier-styles.css)

File: src/app/supplier/products/add/page.js [hunk 7]
```diff
@@ -0,0 +1,408 @@
...
+                  </Upload.Dragger>
+                </Form.Item>
+                <Form.Item
+                  style={{ display: "flex", justifyContent: "flex-end" }}
+                >
+                  <Button
+                    type="primary"
+                    htmlType="submit"
+                    loading={uploading}
+                    style={{ width: 150 }}
```
Comment (lakshankd) [inline src/app/supplier/products/add/page.js:394]: move this to a separate css file (supplier-styles.css).....

File: src/components/supplier/SupplierPageTitle.js [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+const { Title } = Typography;
+
+const SupplierPageTitle = ({ icon, pageTitle }) => {
+  return (
+    <Flex
+      gap="middle"
+      align="center"
+      className="seller-page-title-flex"
+      style={{ height: "100%" }}
```
Comment (lakshankd) [inline src/components/supplier/SupplierPageTitle.js:13]: move this to a separate css file (supplier-styles.css)

File: src/components/supplier/SupplierPageTitle.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    >
+      {icon}
+      <Title level={4} style={{ margin: 0 }}>
```
Comment (lakshankd) [inline src/components/supplier/SupplierPageTitle.js:16]: move this to a separate css file (supplier-styles.css)

**Review by lakshankd — CHANGES_REQUESTED** [review]

File: src/app/supplier/products/add/page.js (hunk 6 above)
Comment (salinda-yasas-99) [reply src/app/supplier/products/add/page.js:159]: check the logic first

File: src/app/supplier/products/add/page.js (hunk 3 above)
Comment (salinda-yasas-99) [reply src/app/supplier/products/add/page.js:91]: sloved

**Review by lakshankd — APPROVED** [review]

**Pull request merged**